src/test/java/com/microsoft/sqlserver/jdbc/SQLServerDriverTest.java

```diff
@@ -190,4 +190,22 @@ public void testConnectionDriver() throws SQLException {
             }
         }
     }
+    
+    /**
+     * test application name
+     * 
+     * @throws SQLException
+     */
+    @Test
+    public void testApplicationName() throws SQLException {
+        try (Connection conn = DriverManager.getConnection(connectionString);
+             Statement stmt = conn.createStatement();
+             ResultSet rs = stmt.executeQuery("SELECT program_name FROM sys.dm_exec_sessions WHERE session_id = @@SPID")) {

```

Comment:
Good to have this test which is actually querying the program name

Reply:
Yes